example/broker.cpp

```diff
@@ -84,7 +116,8 @@ int main(int argc, char **argv) {
 #endif // defined(MQTT_USE_LOG)
             ("certificate", boost::program_options::value<std::string>(), "Certificate file for TLS connections")
             ("private_key", boost::program_options::value<std::string>(), "Private key file for TLS connections")
-        ;
+            ("certificate_reload_interval", boost::program_options::value<unsigned int>()->default_value(3600), "Reload interval for the certificate and private key files (seconds)")
```

Comment:
```suggestion
            ("certificate_reload_interval", boost::program_options::value<unsigned int>()->default_value(0), "Reload interval for the certificate and private key files (seconds)")
```

Never reload by default is better. 0 means no reload.

Reply:
Not all users expect automatic reload. It should be optional functionality.